rb/lib/selenium/webdriver/edge.rb

```diff
@@ -26,23 +26,11 @@ module Selenium
   module WebDriver
     module Edge
       def self.driver_path=(path)
-        warn <<-DEPRECATE.gsub(/\n +| {2,}/, ' ').freeze
```

Comment:
You changed my mind. Made sense to be able to set it once in a `spec_helper.rb` rather than have to pass it in everywhere in the code. Yes, that's how Java does it, but Ruby has more flexibility, so I think we should allow it. Are you disagreeing now? :)